driver-core/src/test/resources/unified-test-format/change-streams/change-streams.json

```diff
@@ -559,6 +559,21 @@
     },
     {
       "description": "Test new structure in ns document MUST NOT err",
+      "runOnRequirements": [
+        {
+          "minServerVersion": "3.6",
+          "topologies": [
+            "replicaset"
+          ]
+        },
+        {
+          "minServerVersion": "3.6",
+          "maxServerVersion": "5.0",
+          "topologies": [
+            "sharded-replicaset"
+          ]
+        }
+      ],
       "operations": [
```

Comment:
This code is different from the one in the spec repo: https://github.com/mongodb/specifications/blob/master/source/change-streams/tests/unified/change-streams.json#L622-L623

Is there a PR pending to be merged there, and the change was taken from that PR, or is it a temporary change that we will have to undo once the latest server is fixed? If it is the latter, can we have a Jira task reminding us about undoing this change?

Reply:
It's a temporary change that should be undone once [SERVER-65497](https://jira.mongodb.org/browse/SERVER-65497) issue is fixed.  Opened [JAVA-4579](https://jira.mongodb.org/browse/JAVA-4579) to track

Reply:
Thanks!